metpy/calc/basic.py

```diff
@@ -220,3 +222,28 @@ def heat_index(temperature, rh, mask_undefined=True):
             hi = units.Quantity(masked_array(hi, mask=mask), hi.units)
 
     return hi
+
+
+@exporter.export
+def pressure_to_height_std(pressure):
+    '''Convert pressure data to heights using the U.S. standard atmosphere.
+
+    Parameters
+    ----------
+    pressure: array_like
+        Atmospheric pressure
+
+    Returns: array_like
```

Comment:
Ah...if you look at the other functions, the returns documentation needs to be its own section:
```
    Returns
    -------
```